contrib/openvpn/files/update-resolv-conf

```diff
@@ -0,0 +1,70 @@
+#!/bin/sh
+
+# Parses DHCP options from openvpn to update resolv.conf
+# To use set as 'up' and 'down' script in your openvpn *.conf:
+# up /etc/openvpn/update-resolv-conf
+# down /etc/openvpn/update-resolv-conf
+#
+# Used snippets of resolvconf script by Thomas Hood <jdthood@yahoo.co.uk>
+# and Chris Hanson
+# Licensed under the GNU GPL.  See /usr/share/common-licenses/GPL.
+# 07/2013 colin@daedrum.net Fixed intet name
+# 05/2006 chlauber@bnc.ch
+#
+# Example envs set from openvpn:
+# foreign_option_1='dhcp-option DNS 193.43.27.132'
+# foreign_option_2='dhcp-option DNS 193.43.27.133'
+# foreign_option_3='dhcp-option DOMAIN be.bnc.ch'
+# foreign_option_4='dhcp-option DOMAIN-SEARCH bnc.local'
+
+## The 'type' builtins will look for file in $PATH variable, so we set the
+## PATH below. You might need to directly set the path to 'resolvconf'
+## manually if it still doesn't work, i.e.
+RESOLVCONF=/usr/bin/resolvconf
+## export PATH=$PATH:/sbin:/usr/sbin:/bin:/usr/bin
+## RESOLVCONF=$(type -p resolvconf)
+
+case $script_type in
+
+up)
+  for option in $(env | grep "foreign_option_" | sed "s/\w\+=//" | tr "\n" " ") ; do
```

Comment:
this doesn't work because changing all newlines to ' ' means instead of looping lines you now loop 1 value at a time, i.e. for foreign_option_3='dhcp-option DOMAIN be.bnc.ch' instead of `option` being `dhcp-option DOMAIN be.bnc.ch` it loops three times with each of those space separated values

you probably want to remove the `tr` entirely and set `IFS=$'\n'` instead

Reply:
or if you don't want to use the new posix extension,

```
IFS='
'
```

Reply:
i trust that you actually tested it because i definitely did not